mindmap/static/js/src/mindmap.js

```diff
@@ -64,13 +59,13 @@ function loadJSMind(callback) {
         callback();
     } else {
         // Load jsMind dynamically using $.getScript
-        $.getScript("https://unpkg.com/jsmind@latest/es6/jsmind.js")
+        $.getScript("https://unpkg.com/jsmind@0.6.4/es6/jsmind.js")
             .done(function () {
                 // Assign jsMind to the window object after it's loaded
                 window.jsMind = jsMind;
 
                 // Load jsMind.draggable-node dynamically using $.getScript
-                $.getScript("https://unpkg.com/jsmind@latest/es6/jsmind.draggable-node.js")
+                $.getScript("https://unpkg.com/jsmind@0.6.4/es6/jsmind.draggable-node.js")
```

Comment:
why changing this?

Reply:
It's not a good practice to use the latest release, I'm just pinning the latest compatible version